cmd/tailscale/backend.go

```diff
@@ -87,16 +87,14 @@ func newBackend(dataDir string, jvm *jni.JVM, store *stateStore, settings settin
 		logID.UnmarshalText([]byte(storedLogID))
 	}
 	b.SetupLogs(dataDir, logID)
-	d, err := dns.NewNoopManager()
-	if err != nil {
-		return nil, err
+	cb := &router.CallbackRouter{
+		SetBoth:  b.setCfg,
+		SplitDNS: false, // TODO: make true
```

Comment:
reference a bug in the TODO with more detail?

Reply:
Done.